Source/WebCore/rendering/TextPainter.cpp

```diff
@@ -129,7 +129,7 @@ void TextPainter::paintTextOrEmphasisMarks(const FontCascade& font, const TextRu
     m_glyphDisplayList = nullptr;
 }
 
-void TextPainter::paintTextWithShadows(const ShadowData* shadow, const FilterOperations* colorFilter, const FontCascade& font, const TextRun& textRun, const FloatRect& boxRect, const FloatPoint& textOrigin,
+void TextPainter::paintTextWithShadows(const RenderStyle& style, const ShadowData* shadow, const FilterOperations* colorFilter, const FontCascade& font, const TextRun& textRun, const FloatRect& boxRect, const FloatPoint& textOrigin,
```

Comment:
Wonder if would be better to RenderStyle& to be a constructor parameter. These argument lists are pretty long.

Reply:
Changed to be a constructor parameter. ✔️